WebDriver conformance changes for Firefox 114 (#27120)
mdn/content: files/en-us/mozilla/firefox/releases/114/index.md

#### WebDriver BiDi

- Added support for the commands `input.performActions` and `input.releaseActions`, which can be used to emulate user input for interacting with elements on web pages. Similar to Marionette all the available input sources of the WebDriver specification are supported, which are `key`, `pointer`, and `wheel` ([Firefox bug 1832380](https://bugzil.la/1832380)).
- Added support for custom browser to client messages, which allows to send a `script.message` event from within a script formerly installed via `script.addPreloadScript` ([Firefox bug 1824187](https://bugzil.la/1824187)).

> - Added the `serializationOptions` parameter for `script.evaluate` and `script.callFunction` to customize the `RemoteValue` serialization ([Firefox bug 1824953](https://bugzil.la/1824953)).

Reviewer: Should we maybe also say here `Added support for the `serializationOptions` parameter`? To align with other entries.

Author: Yes, good catch. I'll get this updated.

- Fixed an issue where both the `script.evaluate` and `script.callFunction` commands did not include the stack trace and failed to properly build the exception details for a rejected Promise ([Firefox bug 1829630](https://bugzil.la/1829630)).
- Fixed an issue where the `browsingContext.domContentLoaded` and `browsingContext.load` events did not report the correct `url`, when the page defined a `<base>` meta tag ([Firefox bug 1825634](https://bugzil.la/1825634)).

#### Marionette

- Fixed an issue where the command `WebDriver:GetComputedRole` didn't properly return the WAI-ARIA roles ([Firefox bug 1822112](https://bugzil.la/1822112)).

> - Fixed an issue where modifier keys are now correctly reset when they are used again within the same `WebDriver:ElementSendKeys` command ([Firefox bug 1776190](https://bugzil.la/1776190)).

Reviewer 2: ```suggestion
- Fixed an issue where modifier keys were not reset when they were used again within the same `WebDriver:ElementSendKeys` command ([Firefox bug 1776190](https://bugzil.la/1776190)).
```

If we say "Fixed an issue where", the rest of the text should describe the issue, not the new behavior? I we want to focus on the new behavior (might be clearer), we can alternatively just drop the "Fixed an issue where", and it would read just fine.

Author: Lets continue for now with the current phrasing given that it's in use for quite a few releases and it was fine. I'll apply your suggestion. Thanks.
